vsphere/README.md

```diff
@@ -85,6 +85,8 @@ See [service_checks.json][12] for a list of service checks provided by this inte
 
 You can limit the number of VMs pulled in with the VMWare integration using the `vsphere.d/conf.yaml` file. See the `resource_filters` parameter section in the [sample vsphere.d/conf.yaml][4].
 
+### Note: Make Sure `collect_tags: true` is set
```

Comment:
This doesn't seem like it should be a heading.

What section does this relate to? Where does this value need to be set?